crates/sui-framework/packages/bridge/sources/treasury.move

```diff
@@ -69,6 +69,21 @@ module bridge::treasury {
         native_token: bool
     }
 
+    public fun token_id<T>(self: &BridgeTreasury): u8 {
+        let metadata = self.get_token_metadata<T>();
+        metadata.id
+    }
+
+    public fun decimal_multiplier<T>(self: &BridgeTreasury): u64 {
+        let metadata = self.get_token_metadata<T>();
+        metadata.decimal_multiplier
+    }
+
+    public fun notional_value<T>(self: &BridgeTreasury): u64 {
+        let metadata = self.get_token_metadata<T>();
+        metadata.notional_value
+    }
```

Comment:
do these need to be `public`? It looks like people won't be able to get `&BridgeTreasury` anyway: https://github.com/MystenLabs/sui/blob/feature/native-bridge/crates/sui-framework/packages/bridge/sources/bridge.move#L674

Reply:
I have not looked if they are used or not as they seemed accessors that are pretty harmless.
I can change them if we want